src/controllers/startOrOpenSaved.controller.js

```diff
@@ -41,16 +43,22 @@ module.exports = class StartOrOpenSavedController extends BaseController {
       // Create new application in Dynamics and set the applicationId in the cookie
       const application = new Application()
       application.statusCode = Dynamics.StatusCode.DRAFT
-      await application.save({ authToken })
+      const permitType = request.query['permit-type']
+      const context = { authToken }
+      await application.save(context)
 
       // Set the application ID in the cookie
       cookie.applicationId = application.id
 
       redirectPath = Routes.BESPOKE_OR_STANDARD_RULES.path
 
+      // Save the permit type in the Data store
+      context.applicationId = application.id
+      // Save the permit type in the Data store
+      await DataStore.save(context, { permitType })
```

Comment:
The permit type from the query only indicates which option should be initially selected on the page, so doesn't constitute a selection by the user.
It would be better to persist this value on the post of the bespokeOrStandardRules controller instead.